web/sites/development.services.yml

```diff
@@ -0,0 +1,6 @@
+# Local development services.
+parameters:
+  http.response.debug_cacheability_headers: true
+services:
+  cache.backend.null:
+    class: Drupal\Core\Cache\NullBackendFactory
```

Comment:
Yeah, but do you mind doing that in a separate PR? This one is just the default "development" services file provided by Drupal Scaffold.